es6-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,53 @@
+import { createElement, noop } from '../lib';
```

Comment:
This is the benefit of importing from the `lib/index.js`. You can import multiple things from the `lib` folder on 1 line.